src/components/WishList/index.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import Property from './Property';
+// import Filter from './Filter';
+
+import { SERVER_URL } from '../../utils/constants';
+
+class WishList extends Component {
+    
+    constructor(props) {
+        super(props);
+        this.state = {
+            loading: true,
+            properties: [],
+            visible: 3
+        }          
+    }
+
+    async componentDidMount() {
+        const url = SERVER_URL + '/property';
```

Comment:
see this URL is going to change we don't have a route for it now , I tell the backend developer to implement it